src/app/debug/table/table-settings-modal/table-settings-modal.component.html

```diff
@@ -102,7 +102,7 @@ <h5>Report table</h5>
         class="fa fa-refresh"></i></button>
       <button title="Revert changes" type="button" class="btn btn-info" (click)="loadSettings()"><i
         class="fa fa-undo"></i></button>
-      <button data-cy-settings-modal-save-changes title="Save changes" type="submit" id="saveTableSettings"
+      <button data-cy-settings-modal='saveChanges' title="Save changes" type="submit" id="saveTableSettings"
```

Comment:
Modal mag je hier weglaten
```suggestion
      <button data-cy-settings='saveChanges' title="Save changes" type="submit" id="saveTableSettings"
```